dss/api/bundles/__init__.py

```diff
@@ -152,6 +152,9 @@ def enumerate(replica: str,
     :param search_after: used to page searches, should not be set by the user.
     """
     if prefix:
+        if any(c.isupper() for c in prefix):
+            sys.stderr.write('Note: prefix contains capital letters!  All data-store UUIDs are lower-cased (even if '
```

Comment:
Would it be better to use `logger.warning()` or `logger.error()` here? 

Also, AFAICT this error message never makes it to the user, it happens on the API server - so is this warning actually necessary?

Reply:
That's a good point, should probably be added to `dcp-cli` instead.

Reply:
There isn't an easy way to add it to the CLI, and I think the description of the argument is explicit enough so I'm tabling this unless I get feedback that it's a must-have in the CLI.